pkg/testutil/require.go

```diff
@@ -598,3 +598,41 @@ func getPodLog(ctx context.Context, podName string, namespace string, clientset
 	}
 	return buf.String(), nil
 }
+
+// RequirePVCToResize checks that the PVCs are resized correctly
+func RequireNumberOfPVCs(t *testing.T, ctx context.Context, sb testenv.DiffingSandbox, b ClusterBuilder, quantity int) {
+	cluster := b.Cluster()
+	var boundPVCCount = 0
+
+	// TODO rewrite this
```

Comment:
Need more details, and also an issue please

Reply:
This is copy paste from the esize PVCs but i'll add an issue to cover both

Reply:
Created #631